docs/getting_started/setup_your_environment.md

```diff
@@ -58,8 +58,8 @@ and then `compinit` to generate them again.
 Example (zsh + oh-my-zsh) [recommended for zsh users] :
 
 ```shell
-mkdir ~/.oh-my-zsh/custom/plugins/deno
-deno completions zsh > ~/.oh-my-zsh/custom/plugins/deno/_deno
+mkdir $ZSH_CUSTOM/deno
```

Comment:
This might be `$ZSH_CUSTOM/plugins/deno`?

Reply:
Oh yeah you are correct